src/components/Checkbox/Checkbox.js

```diff
@@ -142,6 +207,8 @@ Checkbox.propTypes = {
   deselect9: PropTypes.func,
   select10: PropTypes.func,
   deselect10: PropTypes.func,
+  /** @uxpinignoreprop */
+  inline: PropTypes.bool,
```

Comment:
How is this used? The idea is that all the existing prototypes using the inline still render as inline. We can not break existing prototypes

Reply:
added changes